aiida_common_workflows/workflows/eos.py

```diff
@@ -155,7 +155,8 @@ def get_sub_workchain_builder(self, scale_factor, reference_workchain=None):
     def run_init(self):
         """
         Run the first workchain.
-        
+
+
```

Comment:
This should really just be a single newline.

Reply:
Have removed this line.